modules/cmdb/conf/conf.go

```diff
@@ -178,10 +178,11 @@ func Load() {
 	fmt.Println(fileMaxUploadByte.String())
 	cfg.FileMaxUploadSize = fileMaxUploadByte
 
-	OrgWiteList = map[string]bool{
+	OrgWhiteList = map[string]bool{
 		UIDomain():                          true,
 		OpenAPIDomain():                     true,
 		"openapi.default.svc.cluster.local": true,
+		"www.erda.cloud":                    true,
```

Comment:
do not use hard-coded domain, UIDomain is enough

Reply:
Done